tests/test_robot_states_flatcache.py

```diff
@@ -114,3 +115,86 @@ def camera_pose_test(flatcache):
 
 def test_camera_pose_flatcache_on():
     camera_pose_test(True)
+
+
+def test_robot_load_drive():
+    if og.sim is None:
+        # Set global flags
+        gm.ENABLE_OBJECT_STATES = True
+        gm.USE_GPU_DYNAMICS = True
+        gm.ENABLE_TRANSITION_RULES = False
+    else:
+        # Make sure sim is stopped
+        og.sim.stop()
+
+    config = {
+        "scene": {
+            "type": "Scene",
+        },
+    }
+
+    env = og.Environment(configs=config)
+
+    # Iterate over all robots and test their motion
+    for robot_name, robot_cls in REGISTERED_ROBOTS.items():
+        robot = robot_cls(
+            name=robot_name,
+            obs_modalities=[],
+        )
+        env.scene.add_object(robot)
+
+        # At least one step is always needed while sim is playing for any imported object to be fully initialized
+        og.sim.play()
+        og.sim.step()
+
+        # Reset robot and make sure it's not moving
+        robot.reset()
+        robot.keep_still()
+
+        # Set viewer in front facing robot
+        og.sim.viewer_camera.set_position_orientation(
+            position=np.array([2.69918369, -3.63686664, 4.57894564]),
+            orientation=np.array([0.39592411, 0.1348514, 0.29286304, 0.85982]),
+        )
+
+        # If this is a locomotion robot, we want to test driving
+        if isinstance(robot, LocomotionRobot):
+            initial_base_pos = robot.get_position()
+            robot.move_forward(0.1)
```

Comment:
I felt like very people will use `move_forward`. Maybe we should call `env.step(action)` with `base_action_idx` instead?